gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcDirectServerStreamingCallable.java

```diff
@@ -70,4 +78,110 @@ public void serverStreamingCall(
     ClientCall<RequestT, ResponseT> call = GrpcClientCalls.newCall(descriptor, context);
     return ClientCalls.blockingServerStreamingCall(call, request);
   }
+
+  /**
+   * Wraps a GRPC ClientCall in a {@link StreamController}. It feeds events to a {@link
+   * ResponseObserver} and allows for back pressure.
+   */
+  static class GrpcStreamController<RequestT, ResponseT> extends StreamController {
```

Comment:
I'd prefer for the these static package-private classes to be in their own files.

Reply:
done